README.md

```diff
@@ -231,23 +231,7 @@ http.api : ["eth","debug","net"]
 
 ### Beacon Chain (Consensus Layer)
 
-Erigon can be used as an Execution Layer (EL) for Consensus Layer clients (CL). Default configuration is OK. CL
-relies on availability of receipts – don't prune them: don't add character `r` to `--prune` flag. However, old receipts
```

Comment:
Why delete the deposit contract info?

Reply:
Because it's not relevant to most users anymore since `--prune=r` does the right thing by default now.